tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26,py27,py32,py33,py34,pypy,pypy3,coverage,docs
+envlist = py27,py33,py34,pypy,pypy3,coverage,docs
```

Comment:
There's a [testenv:py26] block that could go.

It adds a dependency on ordereddict, but that doesn't seem to be from this package. So probably somewhere else there's going to be some code catching ImportError that can go...